src/pipeline/src/etl/error.rs

```diff
@@ -570,6 +570,18 @@ pub enum Error {
         #[snafu(implicit)]
         location: Location,
     },
+    #[snafu(display("Parse json path error: {}",error.to_string()))]
```

Comment:
```suggestion
    #[snafu(display("Parse json path error"))]
```
We don't need to manually print error, it should be automatically printed by stack.